site.yml

```diff
@@ -91,6 +91,9 @@
       copy: src=jobs dest=/var/lib/jenkins/jobs/default-dsl-job/workspace directory_mode=0750 mode=0640 owner=jenkins group=jenkins
     - name: generate jenkins job from dsl seed job
       shell: "curl -X POST 'http://127.0.0.1:8080/job/default-dsl-job/build' -k $(cat ~/jenkins_auth)"
+    - name: wait for creation and disable terraform jenkins job on vagrant boxes
+      shell: "sleep 15 && curl -X POST 'http://127.0.0.1:8080/job/terraform-deploy/disable' -k $(cat ~/jenkins_auth)"
```

Comment:
@actionjack and I have talked offline about whether we need a unique environment name for each person using the repo. On the one hand, this would avoid errors like one person stomping on another's environment, or the job failing inexplicably. However, we were concerned about a possible proliferation of environments and the associated cost.

We think it's probably more likely than not that it will only be one person/pair using this at a time, so we reckon just one alternative environment. If it starts to cause pain later, we can readdress.